grafana-plugin/src/containers/Rotations/Rotations.helpers.ts

```diff
@@ -37,3 +40,32 @@ export const findColor = (shiftId: Shift['id'], layers: Layer[], overrides?) =>
 
   return color;
 };
+
+export const findClosestUserEvent = (startMoment: dayjs.Dayjs, userPk: User['pk'], layers: Layer[]) => {
+  let minDiff;
+  let closestEvent;
+  if (layers) {
```

Comment:
if you check instead for `if (!layers) return undefined` you'll have 1 less nesting. up to you, personally I like to see as less nesting as possible for if/else/etc.